Earthfile

```diff
@@ -400,6 +403,29 @@ do-verify-headers-impl:
     # by CMake for the VERIFY_INTERFACE_HEADER_SETS target property.
     RUN cmake --build $__build_dir --target all_verify_interface_header_sets
 
+# devdocs :
+#   Builds the developer documentation pages as HTML, and writes the resulting pages into
+#   `_build/devdocs` on the host for browsing.
```

Comment:
Should this be under [`_build/docs/dev`](https://github.com/mongodb/mongo-c-driver/blob/48d0a485c37413ea8caa27e88109715d5e0492dd/docs/dev/Makefile#L17) instead?